src/QMCWaveFunctions/Fermion/SlaterDetBuilder.cpp

```diff
@@ -1059,7 +1059,12 @@ bool SlaterDetBuilder::readDetListH5(xmlNodePtr cur,
     abort();
   }
 
-  if (!hin.push("MultiDet"))
+
+  try
+  {
+    hin.push("MultiDet", false);
+  }
+  catch (...)
   {
     std::cerr << "Could not open Multidet Group in H5 file" << std::endl;
```

Comment:
Is this error handling still needed? There are similar pattern below.

Reply:
I don't know if it's still needed.  This changes the error handling flow the least.

Reply:
I removed the try/catch in this file.  The existing error handling simply aborts on error, and moving to an unhandled exception seems similar.